src/Compilers/Core/Portable/InternalUtilities/Debug.cs

```diff
@@ -13,12 +13,21 @@ namespace Roslyn.Utilities
     internal static class RoslynDebug
     {
         /// <inheritdoc cref="Debug.Assert(bool)"/>
+#if NET9_0_OR_GREATER
+        [OverloadResolutionPriority(-1)]
```

Comment:
Is that really needed? Can't the other overload exist alone?
On .NET, they had to do this for backcompat. But as RoslynDebug is internal, I think it should be safe to just have the right overload?

Reply:
Might not be needed as you say. But I don't want to risk some other DLL depending on this (via InternalsVisibleTo) and then getting MissingMethodExceptions.

Reply:
The way our IVT is setup (no IVT outside our repo) it should be safe to remove this if the repo still compiles.

Reply:
Alright, I will remove the overload resolution priority attribute.